tests/tap.sh

```diff
@@ -42,9 +42,9 @@ for ((i=0;i<${#backends[@]}; ++i)); do
         continue
     fi
 
-    # grep to skip test if backend cannot handle Blocked Restriction
-    if grep -F -q -e 'Backend does not support ElemRestrictionCreateBlocked' \
-            ${output}.err; then
+     # grep to skip test if backend chooses to whitelist test
+    if grep -F -q -e 'Backend does not support' ${output}.err \
+             && grep -F -q -e "$1 $backend" tests/whitelist ; then
```

Comment:
I think it is a safer policy to force backends to explicitly list what functions they do not support so we don't get false passes from the test suite

Reply:
Could we make this something that a backend implements instead of residing in a common whitelist (that will need to be edited and merged)? One solution would be for the backend to implement the function that it "does not support", but raise an error with a different syntax to indicate that it is intentional.